config/config.go

```diff
@@ -105,6 +106,9 @@ type Log struct {
 	Format string `toml:"format" json:"format"`
 	// Disable automatic timestamps in output.
 	DisableTimestamp bool `toml:"disable-timestamp" json:"disable-timestamp"`
+	// DisableErrorVerbose stops annotating logs with the full verbose error
+	// message.
+	DisableErrorVerbose bool `toml:"disable-error-verbose" json:"disable-error-verbose"`
```

Comment:
how about `disable-error-stack`?

Reply:
Maybe better consistent with PD? Actually the field also called `errorVerbose` in zap, the main result of the full output with the stack due to the [`errors` package](https://godoc.org/github.com/pkg/errors#hdr-Formatted_printing_of_errors)

Reply:
Yeah. But I think maybe `disable-error-stack` is easy to be understood by DBAs.

Reply:
ok